src/components/AboutPage/AboutPage.tsx

```diff
@@ -1,54 +1,197 @@
+            title={
+              <a href={repourRepositoryUrl} target="_blank" rel="noopener noreferrer">
+                Repour
+              </a>
+            }
+          >
+            <ServiceContainerLoading {...serviceContainerRepourVersion} variant="inline" title="Repour version">
+              <VersionText
+                version={serviceContainerRepourVersion.data?.version}
+                revision={serviceContainerRepourVersion.data?.commit}
+                repositoryUrl={repourRepositoryUrl}
+              />
+            </ServiceContainerLoading>
+          </AttributesItem>
+
+          <AttributesItem
+            title={
+              <a href={bifrostRepositoryUrl} target="_blank" rel="noopener noreferrer">
+                Bifrost
               </a>
-            </TextListItem>
-            <TextListItem component="dd">
-              {process.env.REACT_APP_VERSION || <EmptyStateSymbol text={false} title="Version" />} (
-              {process.env.REACT_APP_GIT_SHORT_SHA || <EmptyStateSymbol text={false} title="Revision" />})
-            </TextListItem>
-          </TextList>
-        </TextContent>
+            }
+          >
+            <ServiceContainerLoading {...serviceContainerBifrostVersion} variant="inline" title="Bifrost version">
+              <VersionText
+                version={serviceContainerBifrostVersion.data?.version}
+                revision={serviceContainerBifrostVersion.data?.commit}
+                repositoryUrl={bifrostRepositoryUrl}
+              />
+            </ServiceContainerLoading>
+          </AttributesItem>
+
+          <AttributesItem
+            title={
+              <a href={dependencyAnalyzerRepositoryUrl} target="_blank" rel="noopener noreferrer">
+                Dependency Analyzer
+              </a>
+            }
+          >
+            <ServiceContainerLoading
+              {...serviceContainerDependencyAnalyzerVersion}
+              variant="inline"
+              title="Dependency Analyzer version"
+            >
+              <VersionText
+                version={serviceContainerDependencyAnalyzerVersion.data?.version}
+                revision={serviceContainerDependencyAnalyzerVersion.data?.commit}
+                repositoryUrl={dependencyAnalyzerRepositoryUrl}
+              />
+            </ServiceContainerLoading>
+          </AttributesItem>
+        </Attributes>
       </ContentBox>
     </PageLayout>
   );
 };
+
+interface IVersionText {
+  version?: string;
+  revision?: string;
+  repositoryUrl?: string;
+}
+
+const VersionText = ({ version, revision, repositoryUrl }: IVersionText) => (
+  <>
+    {version || <EmptyStateSymbol text={false} title="Version" />} (
+    {(revision && repositoryUrl && (
+      <a href={`${repositoryUrl}/tree/${revision}`} target="_blank" rel="noopener noreferrer">
+        {revision}
+      </a>
+    )) ||
+      revision || <EmptyStateSymbol text={false} title="Revision" />}
+    )
+  </>
+);
```

Comment:
maybe move this to before the main About page component as this is an inner component

Reply:
Yes, I think the main component should be on top of the file because it is more important.